src/apps/myaccount/app.js

```diff
@@ -270,7 +271,7 @@ define(function(require) {
 				if (currentAccount.hasOwnProperty('trial_time_left') && monster.config.api.hasOwnProperty('screwdriver')) {
 					monster.pub('auth.showTrialInfo', currentAccount.trial_time_left);
 				// Only direct them to my account walkthrough if it's not a trial account
-				} else if (self.hasToShowWalkthrough()) {
+				} else if (self.hasToShowWalkthrough() && !self.appFlags.disableFirstUseWalkthrough) {
```

Comment:
Sorry, I just noticed the `hasToShowWalkthrough` function. It would be better to check the flag `self.appFlags.disableFirstUseWalkthrough` inside this function, because it is subscribed to the `myaccount.hasToShowWalkthrough` which means that the condition may be checked also outside of the `myaccount` app.